portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -106,6 +106,29 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     LinkedList<BlogMessage> loadedBlogMessages = blogMap.getMessages(
         tagsToSearch, numberOfCommentsToDisplay);
 
+    //get user email
+    UserService userService = UserServiceFactory.getUserService();
+    String email = (String) userService.getCurrentUser().getEmail();
+
+    //check to see if user follows any tags
+    if (LoadFollowedTags.hasFollowedTags(email)) {
+      LoadFollowedTags loadTags = new LoadFollowedTags();
+      List<String> newTags = loadTags.getFollowedTags(email);
```

Comment:
You can take out line 115 and since getFollowedTags is static, you can call it directly like LoadFollowedTags.getFollowedTags(email)